ui/docs/reference/platform/object/download-methods.mdx

```diff
@@ -55,20 +55,48 @@ For example, a request that can use either the `simple` or `superFancy` methods
 ```
 
 The known methods are defined below.
+See the API definitions for the object service for details on the request and response payloads.
 
 ### `simple` Download Method
 
-The `simple` method implements [simple downloads](/simple-downloads).
+The `simple` method is always supported, as it is used to implement [simple downloads](/simple-downloads).
 The response contains a URL against which a `GET` request can be made to retrieve the object's data.
+This method should only be used by generic clients that are not aware of the particulars of the object service, such as `curl` or web browsers.
+Taskcluster-aware clients should use the `getUrl` method, which is always available.
```

Comment:
@djmitche Can we remove this line, until this is implemented, because I think you wanted to release the server parts before implementing the client parts?

Reply:
I think this line is valid with only this PR landed, though -- callers should prefer the `getUrl` method.  That's just not easy (yet) with the provided libraries.

Reply:
The part I find confusing is "which is always available", as this seems to suggest (to me) that the clients support the `getUrl` method in their native download methods (otherwise I'm not sure what it means).

Perhaps just replace "which is always available" with "where possible"? Or we could add a note to say that clients that wish to download objects may call `object.startDownload` with `{ acceptDownloadMethods: { getUrl: true }}` and then use the returned url to retrieve the object, and that in future, there will be a native helper function/method that will provide this functionality (and maybe cite the method/function name if it is standard across clients?).

Does my point of confusion make sense?

Reply:
It's always available _on the server_, regardless of the object service backend.  The client libraries are just a convenience provided to make it easier to interact with the object service (they are _very_ convenient, especially for the object service), so it's implicit that they would call `startDownload` with `getUrl: true` if they wanted to use this method.

Anyway, this is a tiny, transient detail -- the client libraries will support it soon.  It's hard to see how this would cause anyone any confusion in the interim, since nothing is using the object service.